community-operators/couchbase/couchbase.1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+      mediatype: image/png
+  links:
+    - name: Couchbase
+      url: 'https://www.couchbase.com'
+    - name: Documentation
+      url: 'https://docs.couchbase.com/operator/1.1/overview.html'
+    - name: Downloads
+      url: 'https://www.couchbase.com/downloads'
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    spec:
+      deployments:
+        - name: couchbase-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: couchbase-operator
+            template:
+              metadata:
+                labels:
+                  name: couchbase-operator
+              spec:
+                containers:
+                  - command:
+                      - couchbase-operator
+                    env:
```

Comment:
Since this operator doesn't have the `WATCH_NAMESPACE` env variable, how does it know which namespaces to watch on cluster?

Reply:
It uses `MY_POD_NAMESPACE` to determine which namespace to watch.

Reply:
added the reference to `olm.targetNamespaces` for forward-compatibility